### DISCUSSION TITLE
How to hide taskbar icon in rust winit slint project?

### DISCUSSION THREAD

**Discussion by marsheinrich** [slint-ui/slint#5299] [Q&A]
How to hide taskbar icon when app running?
App should run in several OS.
let window = WindowBuilder::new()
        .with_title("")
        .with_inner_size(LogicalSize::new(100, 100))
        // .with_decorations(false)
        // .with_inner_size(monitor_size)
        .with_always_on_top(true)
        .build(&event_loop)
        .unwrap();
And I hope include slint file and execute.
But I used winit(0.22.0)'s WindowBuilder.
Please help me

**Comment by ogoffart** [comment]
There is no way to do this currently using the Slint API.

But you can access to winit window builder using the private API through the internal i-slint-backend-winit crate https://docs.rs/i-slint-backend-winit

**Comment by felipetesc** [comment]
create a blank icon, remove the title and use decorations ?
